sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -190,6 +192,32 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
   }
 
   /**
+   * Executes the query and calls the {@link org.apache.spark.sql.util.QueryExecutionListener}
+   * methods.
+   *
+   * @param funcName A identifier for the method executing the query
+   * @param qe the @see [[QueryExecution]] object associated with the query
```

Comment:
Could you please fix the doc by following what https://github.com/apache/spark/pull/16013 did?